website-cluster-centos/azuredeploy.json

```diff
@@ -0,0 +1,657 @@
+    "virtualNetworkName": "websiteVNET1",
+    "numDataDisks": "4",
+    "apiVersion": "2015-01-01",
+    "networkSettings": {
+      "virtualNetworkName": "[variables('virtualNetworkName')]",
+      "addressPrefix": "10.0.0.0/24",
+      "subnet": {
+        "web": {
+          "name": "web",
+          "prefix": "10.0.0.0/28",
+          "vnet": "[variables('virtualNetworkName')]"
+        },
+        "mysql": {
+          "name": "mysql",
+          "prefix": "10.0.0.16/28",
+          "vnet": "[variables('virtualNetworkName')]"
+        },
+        "redis": {
+          "name": "redis",
+          "prefix": "10.0.0.32/28",
+          "vnet": "[variables('virtualNetworkName')]"
+        },
+        "haproxy": {
+          "name": "haproxy",
+          "prefix": "10.0.0.48/28",
+          "vnet": "[variables('virtualNetworkName')]"
+        }
+      },
+      "statics": {
+        "clusterRange": {
+          "base": "10.0.0.",
+          "start": 5
+        },
+        "primaryIp": "10.0.0.240"
+      }
+    },
+    "lbOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').lbNodeScript]"
+      ]
+    },
+    "webOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').webNodeScript]"
+      ]
+    },
+    "redisMasterOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').redisMasterNodeScript]"
+      ]
+    },
+    "redisSlaveOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').redisSlaveNodeScript]"
+      ]
+    },
+    "mysqlMasterOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').mysqlMasterNodeScript]"
+      ]
+    },
+    "mysqlSlaveOsSettings": {
+      "imageReference": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "[parameters('centOsVersion')]",
+        "version": "latest"
+      },
+      "scripts": [
+        "[variables('environment').mysqlSlaveNodeScript]"
+      ]
+    }
+  },
+  "resources": [
+    {
+      "name": "shared",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "[variables('apiVersion')]",
```

Comment:
use literal value for apiVersion otherwise JSON editors can't provide intellisense.